scheduler/config/kafka-internal.json

```diff
@@ -1,4 +1,5 @@
 {
+    "topicPrefix":"foo",
```

Comment:
should this be `foo`?  and is there an easier way of setting this up for compose?

Reply:
Good catch - That's a bug from testing - will revert